pom.xml

```diff
@@ -101,6 +101,18 @@
       <artifactId>trilead-api</artifactId>
       <version>1.0.1</version>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>net.i2p.crypto</groupId>
+          <artifactId>eddsa</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>net.i2p.crypto</groupId>
+      <artifactId>eddsa</artifactId>
```

Comment:
should it be a dependency on the sshd-module instead?

Reply:
yep, it should change in trilead